src/pybind/mgr/dashboard/frontend/e2e/rgw/daemons.po.ts

```diff
@@ -1,5 +1,51 @@
+import { $$, by, element } from 'protractor';
 import { PageHelper } from '../page-helper.po';
 
 export class DaemonsPageHelper extends PageHelper {
   pages = { index: '/#/rgw/daemon' };
+
+  checkTables() {
+    this.navigateTo();
+
+    this.getTableCount()
+      .getText()
+      .then((daemoncount) => {
+        // Only check for details table loading if there's at least 1 daemon present
+        if (daemoncount.indexOf('0 total') === -1) {
```

Comment:
`vstart` should always have daemons, right?
if something went wrong and there are no daemons this test will pass.

I would remove the validation and always test this.
```suggestion
```